### PR TITLE
Fixes #1849: Turn the summary tab into an actual summary

### LINKED ISSUE #1849: Turn the summary tab into an actual summary
image

### PR DESCRIPTION
**Pull request by balaji-ramavathu**
## Description
Added changes to display the additives of a product in a cardview in the Summary tab.
Added changes to display the nutrition lights of a product in a cardview in the Summary tab.
fixes #1849

## Screen-shots, if any
actual_summary

### CODE REVIEW

**Comment by balaji-ramavathu** [comment]
@teolemon please review

File: app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java [hunk 1]
```diff
@@ -420,8 +443,60 @@ public void onClick(View textView) {
                     CustomTabsIntent customTabsIntent = CustomTabsHelper.getCustomTabsIntent(getContext(), customTabActivityHelper.getSession());
                     CustomTabActivityHelper.openCustomTab(SummaryProductFragment.this.getActivity(), customTabsIntent, nutritionScoreUri, new WebViewFallback());
                 });
+                nutritionLightsCardView.setVisibility(View.VISIBLE);
+                Nutriments.Nutriment fatNutriment = nutriments.get(Nutriments.FAT);
```
Comment (ripetrescu) [inline app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java]: nutriments can be null. Please add NPE check when used.

Comment (balaji-ramavathu) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java]: okay

Comment (balaji-ramavathu) [reply app/src/main/java/openfoodfacts/github/scrachx/openfood/views/product/summary/SummaryProductFragment.java]: done.

**Comment by teolemon** [comment]
Thanks @balaji-ramavathu for the PR :-)
thanks @ripetrescu for your review

**Pull request merged**